.github/workflows/pre-commit.yml

```diff
@@ -7,5 +7,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - uses: hashicorp/setup-terraform@v3
```

Comment:
I think you should specify `terraform_version:` rather than use the default (`latest`).

Reply:
Great point!

Reply:
Why not use the latest: `~> 1.10.2` where 1.10.2 is the latest.

Reply:
If you are required to use a certain version, add a comment why.

Reply:
Ah, I thought I'd use the version specified in `versions.tf`